src/Api/API.php

```diff
@@ -1027,6 +1027,14 @@ protected function getItem($itemtype, $id, $params = [])
             );
         }
 
+        // Decode HTML
+        if (!$this->returnSanitizedContent()) {
+            $fields = array_map(
+                fn ($f) => is_string($f) ? html_entity_decode($f) : $f,
```

Comment:
We should only revert sanitization process.
```suggestion
                fn ($f) => is_string($f) ? Sanitizer::decodeHtmlSpecialChars($f) : $f,
```